frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -330,6 +350,19 @@ export default class ArenaContestList extends Vue {
     this.currentOrder = ContestOrder.SignedUp;
   }
 
+  filterBySignedUp() {
+    this.currentFilterBySignedUp = this.currentFilterBySignedUp ? false : true;
```

Comment:
```suggestion
  toggleFilterBySignedUp() {
    this.currentFilterBySignedUp = !this.currentFilterBySignedUp;
```

el nombre original sugiere que la función estaría haciendo un filtro, pero en realidad lo que hace es activar/desactivar (toggle) el filtro.